pandas/core/frame.py

```diff
@@ -883,27 +883,53 @@ def dot(self, other):
     @classmethod
     def from_dict(cls, data, orient='columns', dtype=None, columns=None):
         """
-        Construct DataFrame from dict of array-like or dicts
+        Construct DataFrame from dict of array-like or dicts.
+
+        Creates DataFrame object from dictionary by columns or by index
+        allowing dtype specification.
 
         Parameters
         ----------
         data : dict
-            {field : array-like} or {field : dict}
+            Of the form {field : array-like} or {field : dict}.
         orient : {'columns', 'index'}, default 'columns'
             The "orientation" of the data. If the keys of the passed dict
             should be the columns of the resulting DataFrame, pass 'columns'
             (default). Otherwise if the keys should be rows, pass 'index'.
         dtype : dtype, default None
-            Data type to force, otherwise infer
-        columns: list, default None
+            Data type to force, otherwise infer.
+        columns : list, default None
             Column labels to use when orient='index'. Raises a ValueError
             if used with orient='columns'
+            .. versionadded:: 0.23.0.
 
-            .. versionadded:: 0.23.0
+        See Also
+        --------
+        DataFrame.from_records : DataFrame from ndarray (structured dtype),
+                                 list of tuples, dict, or DataFrame
+        DataFrame.from_items : DataFrame from sequence of (key, value) pairs.
```

Comment:
Can you also add a link to simply 'DataFrame', as that can also be used as constructor

Reply:
remove from_items as that is now deprecated.